jira-users-report/Jenkinsfile

```diff
@@ -0,0 +1,41 @@
+@Library('pipeline-library@pull/348/head') _
+
+pipeline {
+  triggers {
+    // Execute every 2 hours
+    cron('H H/2 * * *')
+  }
+  options {
+    // This pipeline takes ~1.5 hour to run
+    timeout(time: 2, unit: 'HOURS')
+    lock(resource: 'infra-reports-jira-users', inversePrecedence: true)
+  }
+  agent {
+    kubernetes {
+      yamlFile 'JenkinsAgentPodTemplate.yaml'
+    }
+  }
+  stages {
+    stage('Generate JIRA Users Report') {
+      environment {
+        JIRA_AUTH = credentials('jiraAuth')
+      }
+      steps {
+        dir('jira-users-report') {
+          sh '''
+          JIRA_AUTH="${JIRA_AUTH_USR}:${JIRA_AUTH_PSW}"
+          bash ./user-report.sh > jira-users-report.json
```

Comment:
I'm confused, I though "jiraAuth" contained "USER:PWD" (cf https://github.com/jenkins-infra/infra-reports/pull/37/files#r845483496)
Why it's different here?

Reply:
good point. Just fixed in a new commit 👍